<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="8" colspan="8">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="5" column="14">
     <widget class="QPushButton" name="pushButton_reload">
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </item>
    <item row="5" column="15">
     <widget class="QPushButton" name="pushButton_close">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
    <item row="5" column="13">
     <widget class="QPushButton" name="pushButton_createRequest">
      <property name="text">
       <string>Создать заявку</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
